ci/downstream/.github/workflows/sync.yml

```diff
@@ -26,7 +26,7 @@ jobs:
   sync:
     strategy:
       matrix:
-        branch: ["release-1.8.x", "release-1.9.x", "release-1.10.x"]
+        branch: ["release-1.8.x", "release-1.9.x", "release-1.10.x", "release-1.11.x"]
```

Comment:
We may remove "release-1.9.x" as we want to support only one non LTS release (in this case 1.11).